akka-stream/src/main/scala/akka/stream/impl/io/FileSubscriber.scala

```diff
@@ -28,17 +32,13 @@ private[akka] class FileSubscriber(f: File, bytesWrittenPromise: Promise[Long],
 
   override protected val requestStrategy = WatermarkRequestStrategy(highWatermark = bufSize)
 
-  private var raf: RandomAccessFile = _
   private var chan: FileChannel = _
 
   private var bytesWritten: Long = 0
 
   override def preStart(): Unit = try {
-    raf = new RandomAccessFile(f, "rw") // best way to express this in JDK6, OpenOption are available since JDK7
-    chan = raf.getChannel
-
-    // manually supporting appending to files - in Java 7 we could use OpenModes: FileChannel.open(f, openOptions.asJava)
```

Comment:
Or maybe it doesn't... since chattr could be used hm to restrict "only append". Let's just set append if it works for you and we'll know about linux on jenkins